ouroboros-consensus/src/Ouroboros/Consensus/Node.hs

```diff
@@ -117,7 +118,10 @@ import           Ouroboros.Consensus.Storage.VolatileDB
 -- | Arguments expected from any invocation of 'runWith'
 data RunNodeArgs m addrNTN addrNTC blk = RunNodeArgs {
       -- | Consensus tracers
```

Comment:
Shouldn't this be retitled to `Consensus counters`?

Reply:
Fixed.  Thanks!